doorstop/core/tree.py

```diff
@@ -437,7 +438,12 @@ def get_issues(self, skip=None, document_hook=None, item_hook=None):
         if not documents:
             yield DoorstopWarning("no documents")
         # Check each document
-        for document in documents:
+        trees = [self]
```

Comment:
What is this logic doing? I'm a bit confused to see a variable named `trees` -- is there more than one?